### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/FlashpointProject-launcher/commit/e236ac0aa7fac2ab2900b4851b20217a5a79ef2b#commitcomment-45607142

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/FlashpointProject-launcher/commit/49b61db0a983adb315e376e9f06477a933ed5ca2

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

---

I understand you have preferences for how a final commit should be structured. As I expect a PR to be a discussion about which pieces to include, I prefer to finalize the format at the end once the content is accepted.

### CODE REVIEW

**Comment by colin969** [comment]
A lot of these changes are changing variable names. Please go over them because they shouldn't be commited.

**Comment by jsoref** [comment]
Are the ones that rename functions ok?

File: lang/en.json [hunk 1]
```diff
@@ -350,7 +350,7 @@
     "nonExistingLibrary": "'Library' is not the name of an existing library! (Default will be used)",
     "nonContentFolders": "Non-Content folders found in Curation folder. Are they supposed to be there?",
     "ilc_notHttp": "Use HTTP.",
-    "ilc_nonExistant": "Point to an existing file in your curation's 'content' folder."
+    "ilc_nonExistent": "Point to an existing file in your curation's 'content' folder."
```
Comment (colin969) [inline lang/en.json:353]: Like here is bad.

File: src/renderer/components/pages/ConfigPage.tsx [hunk 2]
```diff
@@ -330,7 +330,7 @@ export class ConfigPage extends React.Component<ConfigPageProps, ConfigPageState
             onChange={(event) => this.onAppPathOverridePathChange(index, event.target.value)}
             text={item.path} />
           <div
-            className='setting__row__content--override-row__seperator'>
+            className='setting__row__content--override-row__separator'>
```
Comment (colin969) [inline src/renderer/components/pages/ConfigPage.tsx:333]: CSS rules shouldn't be changed for compatability reasons.

File: static/window/styles/core.css [hunk 3]
```diff
@@ -2111,7 +2111,7 @@ body {
   padding: 0 1rem;
   align-items: center;
 }
-.setting__row__content--override-row__seperator{
+.setting__row__content--override-row__separator{
```
Comment (colin969) [inline static/window/styles/core.css:2114]: Other side of the usage, CSS rules shouldn't be changed for compatability reasons.

**Comment by colin969** [comment]
Left you comments.

**Comment by colin969** [comment]
Also give it another run over, you were working against Master when you should've been working against Develop, changed the base of the PR.

**Comment by jsoref** [comment]
I've moved these five to the top:
52fa8b2
3264f54
28e6884
563a506
24140d4

I'm going to lop them off in a moment.

**Comment by colin969** [comment]
Seems good, can you squash them down before I merge?

**Comment by jsoref** [comment]
@colin969: done

**Pull request merged**